src/assets/i18n/nl.json

```diff
@@ -291,5 +291,19 @@
             "back": "Terug",
             "import": "Importeer"
         }
-    }
+    },
+    "password-strength-component": {
+      "toggle-advanced": "Geavanceerd wachtwoord info schakelen",
+      "feedback": "Terugkoppeling",
+      "time-to-crack": "Benodigde tijd om je wachtwoord te kraken",
+      "online-throttling": "Online throttling",
```

Comment:
throttling is volgens mij geen Nederlands woord. Is 'afknijpen' niet de Nederlandse term?